<ui version="4.0" >
 <author></author>
 <comment>*********************************************************************
**
** Copyright (C) 1992-$THISYEAR$ $TROLLTECH$. All rights reserved.
**
** This file is part of the $MODULE$ of the Qt Toolkit.
**
** $TROLLTECH_DUAL_LICENSE$
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <exportmacro></exportmacro>
 <class>QtBrushEditor</class>
 <widget class="QWidget" name="QtBrushEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4" >
        <widget class="QToolButton" name="gradientToolButton" >
         <property name="toolTip" >
          <string>Edit Gradient</string>
         </property>
         <property name="text" >
          <string>Gradient...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="5" >
        <widget class="QGroupBox" name="previewBox" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>3</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Preview</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QtBrushWidget" name="brushWidget" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>3</hsizetype>
              <vsizetype>3</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QToolButton" name="patternToolButton" >
         <property name="toolTip" >
          <string>Edit Color Pattern</string>
         </property>
         <property name="text" >
          <string>Color Pattern...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QToolButton" name="textureToolButton" >
         <property name="toolTip" >
          <string>Edit Texture</string>
         </property>
         <property name="text" >
          <string>Texture...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QToolButton" name="editToolButton" >
         <property name="text" >
          <string>Edit...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" colspan="4" >
        <widget class="QListWidget" name="listWidget" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>160</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QToolButton" name="removeButton" >
         <property name="toolTip" >
          <string>Remove From Custom</string>
         </property>
         <property name="text" >
          <string>Remove From Custom</string>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>92</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" >
        <widget class="QToolButton" name="addToCustomButton" >
         <property name="toolTip" >
          <string>Add To Custom</string>
         </property>
         <property name="text" >
          <string>Add To Custom</string>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QToolButton" name="applyButton" >
         <property name="toolTip" >
          <string>Apply Custom</string>
         </property>
         <property name="text" >
          <string>Apply</string>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QtBrushWidget</class>
   <extends>QFrame</extends>
   <header>qtbrushwidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>patternToolButton</tabstop>
  <tabstop>textureToolButton</tabstop>
  <tabstop>gradientToolButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
